### ISSUE TITLE
Change assistant in chat

### ISSUE THREAD

**Issue by dannys42** [BoltAI/BoltAI#222]
**Is your feature request related to a problem? Please describe.**

Somehow my assistant selection did not stick when I created a new chat.  While you can change the system instruction manually, there does not appear to be a way to set or change the assistant.

**Describe the solution you'd like**

In the Inspector, where "System Instruction" is given.  It would be nice to have either a drop-down for assistant selection, or "Custom" where you can enter adhoc text.

The current implementation has the unfortunate effect where if you select "Edit" on a chat that has an Assistant associated, the text is not detached from the assistant definition.  Meaning if I want to refine the assistant, I now have to copy & paste between the chat and the assistant editor.

By having the drop-down, you would solve this problem by allowing the user to choose at any time whether they want the chat to defer to the defined assistants or adhoc system instruction.

**Describe alternatives you've considered**

Can't think of any.

**Additional context**

**Comment by longseespace** [comment]
Good idea. Though you can set an Assistant as default in Settings > Assistant.
